docs/userguide/recovery_device.rst

```diff
@@ -0,0 +1,80 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_userguide_recovery_device:
+
+===============
+Recover devices
```

Comment:
I feel it makes more sense putting the emphasis on the creation of recovery devices so:
```suggestion
Create recovery devices
```
Or
```suggestion
Recovery devices
```

Reply:
The documentation is not only about creating a recovery device and that would make a repetition with the chapter that follow.

We choose to remain consistent with the other titles: verbs + subject

cf: https://github.com/Scille/parsec-cloud/pull/5655#discussion_r1381389161

Reply:
Both "Recover devices" and "Create recovery devices" seem fine to mee (and more or less matches the "verb + subject" too). Maybe "Create recovery devices" is a little bit more clearer for the user.

Reply:
I don't want to use `Create recovery devices` as it repeat with the following chapter `Create a recovery devices` and this documentation is also about using a recovery devices.

I would prefer such title that include both the creation and usage:

```suggestion
Create and use a recovery device
```